docs/DafnyRef/Refinement.md

```diff
@@ -92,32 +92,133 @@ TODO
 
 ## 21.4. Const declarations
 
-A parent `const` declaration may be refined by a refining `const` declaration
-if
+A child `const` declaration may refine a `const` declaration from a
+parent module if
 
 * the parent has no initialization,
 * the child has the same type as the parent, and
 * one or both of the following holds:
    * the child has an initializing expression
-   * the child is declared `ghost` and the parent is not `ghost`, or vice versa
+   * the child is declared `ghost` and the parent is not `ghost`, or vice versa.
 
-To indicate it is a refining declaration, a refining `const` declaration
-contains an ellipsis in this syntax:
-```grammar
-"const" { Attribute } CIdentType "..." [ ":=" Expression ]
-```
```

Comment:
Will it be clear what syntax is used for refining?

Reply:
I partly removed this because it seemed wrong, since I'd never seen an example of `...` used with `const`. I experimented, though, and it does work. So I'll replace this with an example instead of a grammar.